### PR TITLE
Update script to run on prod env where we dont have access to SC/MC or shard_id

### PR DESCRIPTION
**Pull request by morenod**
- ShardID as option argument, on prod we dont need it
- SC/MC timing of namespace creation will be collected only if we have access to them
- Remove default rosa environment because prod didnt work anymore, in the future, we will use --rosa-env staging for stage and nothing for production

### CODE REVIEW

File: rosa-hypershift/rosa-hosted-wrapper.py [hunk 1]
```diff
@@ -1604,3 +1610,4 @@ def main():
 
 if __name__ == '__main__':
     sys.exit(main())
+
```
Comment (krishvoor) [inline rosa-hypershift/rosa-hosted-wrapper.py]: Minor nit, can you kindly remove this space?

**Review by krishvoor — APPROVED** [review]
minor nit, otherwise /lgtm

File: rosa-hypershift/rosa-hosted-wrapper.py (hunk 1 above)
Comment (morenod) [reply rosa-hypershift/rosa-hosted-wrapper.py]: removed, thanks!

**Review by krishvoor — APPROVED** [review]
/lgtm

**Pull request merged**